src/main/java/seedu/address/model/task/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.task.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified tag.
+ */
+public class TaskNotFoundException extends RuntimeException {}
```

Comment:
Can consider making this an checked exception (i.e. extend Exception class) instead since it's a recoverable error.